readthedocs/projects/models.py

```diff
@@ -1089,7 +1089,7 @@ class ImportedFile(models.Model):
     )
     name = models.CharField(_('Name'), max_length=255)
     slug = models.SlugField(_('Slug'))
-    path = models.CharField(_('Path'), max_length=255)
+    path = models.CharField(_('Path'), max_length=4096)
```

Comment:
We should add a comment why 4096 is our magic number here

Reply:
Thank you.
I have added the comment and pushed the changes.